{% extends "base.html" %}
{% block content %}
<h2>Work Experience</h2>

<div class="job">
    <h3>Research Analyst</h3>
    <h4>Department of Surgery, Duke University Medical Center <span class="date">June 2022 – Present</span></h4>
    <ul>
        <li>Python-Based Genomic Analysis Pipeline Development: Developed a robust data analysis pipeline in Python for the detection of overlapping chromosome regions with consistent gains or losses, comparing 13 invasive breast cancer samples with their corresponding non-invasive counterparts. This pipeline was instrumental in uncovering shared genetic aberrations, providing valuable insights into the progression of breast cancer.</li>
        <li>Comparative Genomic Hybridization (CGH) and Copy Number Variation (CNV) Analysis: Conducted CGH-based CNV analysis on invasive and non-invasive breast cancer samples, using the TCGA dataset to obtain a comprehensive profile of genomic alterations and their clinical implications.</li>
        <li>Clinical Data Management and Database Proficiency: Skilled in querying clinical data and managing databases using Epic Maestro Care, Duke Enterprise Data Unified Content Explorer (DEDUCE), and RedCap, within the Duke Health System for streamlined data integration and research.</li>
        <li>HER2-Positive Breast Cancer Database for Neoadjuvant Therapy: Constructed a detailed database of HER2-positive breast cancer patients undergoing neoadjuvant therapy by querying patient health information within the Protected Analytics Computing Environment (PACE), supporting clinical research and outcomes analysis.</li>
        <li>Phase III COMET Trial Database Management: Built and maintained comprehensive databases for the phase III COMET trial on low-risk ductal carcinoma in situ (DCIS), including responsibilities for downstream sample processing and coordination of sample shipment logistics.</li>
        <li>African American Cancer Epidemiology Study (AACES) Contribution: Played a pivotal role in the AACES through microdissection, DNA/RNA extraction, and the application of next-generation sequencing (NGS) technologies, advancing research on cancer epidemiology in underrepresented populations.</li>
    </ul>
</div>

<div class="job">
    <h3>Research Technician II</h3>
    <h4>Pediatric Oncology, Washington University in St. Louis <span class="date">October 2020 – May 2022</span></h4>
    <ul>
        <li>Collaborated across departments at Washington University in St. Louis (e.g., Animal Center, Imaging Center) and companies (e.g.,TransnetYX) to streamline all time sensitive laboratory work from mouse harvesting to data processing. </li>
        <li>Conducted statistical analysis (Prism GraphPad), and IHC image analysis (Imaris, Image J, and NDP View2) to investigate cell proliferation and differentiation in murine models of the Neurofibromatosis type 1 pediatric brain tumor predisposition syndrome.</li>
        <li>Oversaw laboratory operations, including training new personnel in biological experiments (PCR, genotyping, mouse brain dissection), managing mouse colonies, conducting molecular biological experiments, and performing image and data analysis.</li>
        <li>Maintained lab supplies ordering, lab safety, chemical inventory, and equipment maintenance.</li>
    </ul>
</div>

<div class="job">
    <h3>Graduate Researcher</h3>
    <h4>Department of Biochemistry, Wake Forest University <span class="date">August 2018 – May 2020</span></h4>
    <h5>Master’s thesis project: “Susceptibility of Glioblastoma Cells to Oncolytic Vesicular Stomatitis Virus (VSV)”.</h5>
    <ul>
        <li>Analyzed RNA-Seq data (GSEA) to access glioblastoma cell responses to oncolytic viral infection and evaluated protein expression and activation using Western Blots and T tests.</li>
        <li>Conducted in vivo and in vitro experiments, including mouse handling, intracranial injection of oncolytic VSV, and maintaining patient-derived glioblastoma cell lines and virus stocks.</li>
    </ul>
</div>

{% endblock %}
